tests/Integration/Utility/AdapterSecurityAdmin.php

```diff
@@ -28,24 +28,13 @@
 
 namespace Tests\Integration\Utility;
 
-use PrestaShop\PrestaShop\Adapter\LegacyContext;
 use Symfony\Component\HttpKernel\Event\RequestEvent;
-use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
-use Symfony\Component\Security\Core\Authentication\Token\UsernamePasswordToken;
-use Symfony\Component\Security\Core\User\UserProviderInterface;
 
 /**
  * Admin Middleware security
  */
 class AdapterSecurityAdmin
```

Comment:
You should add a comment here to explain that this listener aims at disabling the behaviour from the `Admin` listener in test environment